src/payment/checkout/payment-form/utils/form-validators.js

```diff
@@ -0,0 +1,93 @@
+import getStates from './countryStatesMap';
+
+// eslint-disable-next-line import/prefer-default-export
+export function isPostalCodeRequired(selectedCountry) {
+  const countryListRequiredPostalCode = ['CA', 'GB', 'US'];
+  const postalCodeRequired = countryListRequiredPostalCode.includes(selectedCountry);
+
+  return postalCodeRequired;
+}
+
+export function getRequiredFields(fieldValues, isBulkOrder = false, enableStripePaymentProcessor = false) {
```

Comment:
I added `isBulkOrder` and `enableStripePaymentProcessor` as optional, but the 2/3 places I use this function, I do need to pass in the values. The other place I use it is in the tests where I didn't want to define and pass in these args.